specification/cognitiveservices/OpenAI.Inference/models/completions.create.tsp

```diff
@@ -159,6 +161,14 @@ model Completions {
   @projectedName("csharp", "InternalCreatedSecondsAfterUnixEpoch")
   created: int32;
 
+  @doc("""
+  Content filtering results for zero or more prompts in the request. In a streaming request, 
+  results for different prompts may arrive at different times or in different orders.
+  """)
+  @added(ServiceApiVersions.v2023_06_01_Preview)
+  @projectedName("json", "prompt_annotations")
+  promptFilterResults?: PromptFilterResult[];
```

Comment:
I have a question about this name. Found out we have mostly aligned the naming, which uses the same words in projectName and the property name.

For example,
```
  @projectedName("json", "content_filter_results")
  contentFilterResults?: ContentFilterResults;
```

which contentFilterResults = content_filter_results.

Is this an exception mismatch, `promptFilterResults` and `promp_annotations`?
Is that because "prompt_annotations" is the service-defined name but we want to go with `promptFilterResults` for all SDK languages?

Reply:
That's a great point! I'm not sure if Brandon made this deviation intentionally or not, but I *do* like the idea of us having parallelism between "prompt filter results" and "content filter results" (or, conversely, "prompt [filter] annotations" and "content [filter] annotations") even if the parallelism doesn't exist in REST. It just feels disjoint for us to have different vocabulary in two places that are so close to each other.

We're going to need to do a holistic pass for service vs. client truth across the board (@johanste has rightly raised this a few times) and I'd like to defer any fixup we do here (along with agreed-upon client projection deviation, if we so agree upon it) for that broader pass.